lib/providers/notifications.dart

```diff
@@ -12,7 +14,7 @@ final notificationServiceProviderOverride = notificationServiceProvider.override
   }
   final service = FcmNotificationService(
     onAppOpenWhenNotificationReceived: null,
-    localNotificationService: ref.read(localNotificationServiceProvider),
+    localNotificationService: Platform.isAndroid ? ref.read(localNotificationServiceProvider) : null,
```

Comment:
This is to prevent the native notification prompt to appear on iOS when the notification service starts